src/main/java/uk/gov/hmcts/reform/sscs/jms/listener/HmcHearingsEventTopicListener.java

```diff
@@ -73,7 +75,11 @@ public void onMessage(JmsBytesMessage message) throws JMSException, HmcEventProc
                     hearingId
                 );
 
-                processHmcMessageService.processEventMessage(hmcMessage);
+                if (processEventMessageV2Enabled) {
+                    processHmcMessageService.processEventMessageV2(hmcMessage);
+                } else {
+                    processHmcMessageService.processEventMessage(hmcMessage);
+                }
             }
         } catch (JsonProcessingException | CaseException | MessageProcessingException ex) {
             throw new HmcEventProcessingException(String.format(
```

Comment:
I suppose we need to catch RuntimeExcpetion as well which is throw from ProcessHmcMessageService